src/compressor/lz4/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# lz4
+
+set(lz4_sources
+  CompressionPluginLZ4.cc
+)
+
+add_library(ceph_lz4 SHARED ${lz4_sources})
+add_dependencies(ceph_lz4 ${CMAKE_SOURCE_DIR}/src/ceph_ver.h)
+target_link_libraries(ceph_lz4 ${LZ4_LIBRARY})
+set_target_properties(ceph_lz4 PROPERTIES
+  VERSION 2.0.0
```

Comment:
why the soversion starts at 2.0? and i am wondering if we need to version plugin at all?